### PR TITLE
refactor: chip

### PR DESCRIPTION
**Pull request by thelukewalton**

### CODE REVIEW

**Comment by ahmed-osman3** [comment]
Input chip has no hover color change for active chips
image

**Comment by ahmed-osman3** [comment]
Not sure if there should be a border for active chips
image
image

**Comment by thelukewalton** [comment]
> Not sure if there should be a border for active chips image image

I missed that - will update now

File: lib/src/components/chips/chip.dart [hunk 1]
```diff
@@ -51,127 +41,171 @@ class ZetaChip extends StatefulWidget {
...
             ),
-        ].divide(const SizedBox.square(dimension: ZetaSpacing.small)).toList(),
-      ),
+            decoration: BoxDecoration(
+              color: () {
+                if (states.contains(WidgetState.disabled)) {
+                  return colors.surfaceDisabled;
+                }
+                if (selected) {
+                  return colors.cool.shade90;
```
Comment (ahmed-osman3) [inline lib/src/components/chips/chip.dart:157]: Selected shade feels a bit off. I think it should be colors.surfaceDefaultInverse

File: lib/src/components/chips/filter_chip.dart [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
+///
+/// The chips are commonly used within a [ZetaFilterSelection].
+///
+/// These chips use [Draggable] and can be dragged around the screen and placed in new locations using [DragTarget].
 ///
 /// Extends [ZetaChip].
 class ZetaFilterChip extends ZetaChip {
-  /// Creates a [ZetaInputChip].
+  /// Creates a [ZetaFilterChip].
   const ZetaFilterChip({
```
Comment (mikecoomber) [inline lib/src/components/chips/filter_chip.dart:14]: On the filter chip the text isn't centered when it's unselected

**Review by ahmed-osman3 — APPROVED** [review]

**Review by mikecoomber — APPROVED** [review]

**Pull request merged**